### DISCUSSION TITLE
Subfolder installation

### DISCUSSION THREAD

**Discussion by Caseit18** [HiEventsDev/Hi.Events#295] [General]
Does Hi.Events support installation as a subfolder?
After trouble with bug #253 (or something with a similar result), I tried another ticket product, spent far too much time beating on it, only to discover it doesn't work with subfolders, it needs to be the 'root' website of a domain. So I'd like to figure that out before I spend too much more time on my problem.

**Comment by daveearley** [comment]
There’s no reason it shouldn’t work in a subfolder. However, there’s no specific guide available for this, and it’s unlikely one will be created soon unless someone contributes it.

The app is straightforward—it’s a Laravel API with a React frontend. If you’re familiar with NGINX and Laravel, setting it up should be relatively simple.

**Reply by Caseit18** [reply]
Any reason I can't use apache httpd with a proxy to the nginx ports to handle my SSL? (obviously documentation & support is a DIY).

**Reply by daveearley** [reply]
No reason what that wouldn't work. Any web server should work in theory.
